ui/frontend/gist.js

```diff
@@ -14,11 +14,17 @@ const FILENAME = 'playground.rs';
 export function load(id) {
   return fetch(`${baseUrlStr}/${id}`)
     .then(response => response.json())
-    .then(gist => ({
-      id: id,
-      url: gist.html_url,
-      code: gist.files[FILENAME].content,
-    }));
+    .then(gist => {
+        let filename = FILENAME;
+        for (let x in gist.files) {
+            filename = x;
+        }
+        return {
+            id: id,
+            url: gist.html_url,
+            code: gist.files[filename].content,
+        };
```

Comment:
> With this fix the UI will load the last/only file inside a gist so any name can be chosen.

How about we go ahead and change the code to reflect that reality? In that case, we could just use 

```js
code: gist.files.pop().content,
```

without doing the extra looping through the files and the unused initial setting of the file.

You should also address the linting issues.

---

If you wanted to be really advanced and fully solve #135...

I'd check to see how many files there are. If there's only one, just use that. If there are multiple, I'd combine them all, adding a comment before each one with the name of the file.